packages/katello/rubygem-pulp_docker_client/rubygem-pulp_docker.spec

```diff
@@ -0,0 +1,93 @@
+# Generated from pulp_docker_client-4.0.0b6.dev01562331743.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name pulp_docker_client
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 4.0.0b6.dev01562331743
+Release: 1%{?dist}
+Summary: Pulp 3 Docker API Ruby Gem
+Group: Development/Languages
+License: GPLv2
+URL: https://github.com/pulp/pulp_docker/
```

Comment:
This is the Pulp docker plugin itself. Would https://rubygems.org/gems/pulp_docker_client be a better URL?

Reply:
if they ever get ruby docs published, it may be it.  Will change it for now.